.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x, 22.x]
```

Comment:
i assume this is a safe change as internal codebases are on v20+ already

Reply:
Yeah, and with the engines added, it's now formal.